lib/common/digest.c

```diff
@@ -267,7 +267,8 @@ crm_md5sum(const char *buffer)
     char *digest = NULL;
     gchar *raw_digest = NULL;
 
-    if (buffer == NULL) {
+    // GLib throws an error and returns NULL if buffer is NULL or empty
+    if (pcmk__str_empty(buffer)) {
```

Comment:
Bah, this *might* be worth backporting to 3.0 at least (not planning on any more 2.1 releases).  Are you actually seeing this show up in testing, or just being careful?

Reply:
Just being careful. It's something I noticed by inspection when cleaning up the cibadmin functions.